src/redux/books/books.js

```diff
@@ -0,0 +1,37 @@
+export const ADD_BOOK = 'addBook';
+export const REMOVE_BOOK = 'removeBook';
```

Comment:
- [ ] The value of each action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux) 😉